webpack/scenes/Subscriptions/components/SubscriptionsTable/__tests__/__snapshots__/SubscriptionsTable.test.js.snap

```diff
@@ -192,7 +192,7 @@ Array [
       </tbody>
     </table>
     <form
-      class="content-view-pf-pagination table-view-pf-pagination clearfix"
+      class="content-view-pf-pagination table-view-pf-pagination clearfix "
```

Comment:
This will cause you're tests to fail in CI.  I think there is some package difference between what is on jenkins and what we install locally.  It seems to have to do with patternfly-react but I haven't had a chance to track it down yet.